qbft/messages_encoding.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by fastssz. DO NOT EDIT.
-// Hash: 6732475e30a3885e05ad0c04dfd8b3ca58aae4ebd22fa32781bb39c9d2f307c2
+// Hash: 578300ed5a3dbea99256e5e83e87b91e991213f4bd974c40387c8843a8cdd3d8
```

Comment:
why did this change?
we must be aware of not creating accidental forks

Reply:
I think it didn't do this before (maybe some lib update caused this)
Since I saw no generated code change it seems fine